updated_main.py

```diff
@@ -31,41 +31,26 @@ def discord_bot():
     upper_limit = 96.50
     lower_limit = 96.40
     
-    if current_price >= upper_limit:
-        high_price_alert = (
-                "ALERT HIGH PRICE: Current price of 1 British Pound is = "
-                + str(current_price)
-                + " Indian Rupees"
-            )
-        hook.send(high_price_alert)
+    if getting_current_price.current_price >= upper_limit:
+        hook.send(f"ALERT HIGH PRICE: Current price of 1 British Pound is = {str(getting_current_price.current_price)} Indian Rupees")
         upper_limit += 0.50
 
-    elif current_price <= lower_limit:
-        low_price_alert = (
-                "ALERT LOW PRICE: Current price of 1 British Pound is = "
-                + str(current_price)
-                + " Indian Rupees"
-            )
-        hook.send(low_price_alert)
+    elif getting_current_price.current_price <= lower_limit:
+        hook.send(f"ALERT LOW PRICE: Current price of 1 British Pound is = {str(getting_current_price.current_price)} Indian Rupees")
         lower_limit -= 0.50
 
     else:
-        current_price_alert = (
-                "Current price of 1 British Pound is = "
-                + str(current_price)
-                + " Indian Rupees"
-            )
-        hook.send(current_price_alert)
+        hook.send(f"Current price of 1 British Pound is = {str(getting_current_price.current_price)} Indian Rupees")
             
             
 def send_email():
     """#Sneding automated email"""
     sender = 'mohitpatni0731@gmail.com'
     password = 'ckyryrdajffgmnde'
```

Comment:
Is this real password of mohitpatni0731@gmail.com?

Reply:
No this is the password generated from the security section of the google account which lets the script automatically login to it.

But for the security purpose of the account, I've changed it to inputable format.
Also changed the Discord webhook URL to inputable format.

Reply:
Yeah taking input is correct approach for this.

But why do I still see sender and password hardcoded here?

Reply:
I've changed it to user input already